core/server/data/migrations/versions/2.17/1-canonical-post.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird'),
```

Comment:
> 1-canonical-post.js

Depends which PR get's merged earlier, see https://github.com/TryGhost/Ghost/pull/10585

Reply:
Will make this one `2-*` as `settings` one looks closer to land in master :)